app/views/helpers/feed/update.phtml

```diff
@@ -54,7 +54,7 @@
 			<label class="group-name" for="url"><?= _t('sub.feed.url') ?></label>
 			<div class="group-controls">
 				<div class="stick">
-					<input type="text" name="url" id="url" class="extend" value="<?= $this->feed->url() ?>" />
+					<input type="url" name="url" id="url" class="extend" value="<?= $this->feed->url() ?>" required />
```

Comment:
```suggestion
					<input type="url" name="url" id="url" class="extend" value="<?= $this->feed->url() ?>" required="required" />
```

Reply:
Not needed for HTML5. Do we follow XHTML? I would like it too

Reply:
We have been using the strict syntax as much as possible, kind of XHTML, yes